MdePkg/Include/Library/FdtLib.h

```diff
@@ -247,6 +247,37 @@ FdtSubnodeOffsetNameLen (
   IN INT32        NameLength
   );
 
+/**

```

Comment:
In this commit message, is it really ACPI defines such reserved memory ranges descriptors? Or is it DeviceTree spec?

Reply:
Typo. Fixed it.

Reply:
resolved